src/main/java/wooteco/subway/exception/ExceptionAdvice.java

```diff
@@ -0,0 +1,52 @@
+package wooteco.subway.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+import java.util.Objects;
+
+@RestControllerAdvice
+public class ExceptionAdvice {
+
+    @ExceptionHandler(DuplicatedException.class)
```

Comment:
```suggestion
@ExceptionHandler({DuplicatedException.class, InvalidInsertException.class})
```
ExceptionHandler는 하나의 예외만이 아닌 여러개의 예외를 멀티로 받을 수도 있습니다. 그래서 위처럼 처리 방법이 동일한건 하나의 메서드로 묶어 줄 수 있습니다. 대신 파라미터도 그에 맞춰 변경이 필요할 겁니다.

```java
@Target(ElementType.METHOD)
@Retention(RetentionPolicy.RUNTIME)
@Documented
public @interface ExceptionHandler {

	/**
	 * Exceptions handled by the annotated method. If empty, will default to any
	 * exceptions listed in the method argument list.
	 */
	Class<? extends Throwable>[] value() default {};

}
```